stock_no_extra_move/wizard/stock_transfer_details.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Julien Coux (Camptocamp SA)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, exceptions, models, _
+
+
+class StockTransferDetailsItems(models.TransientModel):
+    _inherit = 'stock.transfer_details_items'
+
+    @api.one
+    @api.constrains('quantity', 'orig_quantity')
+    def _check_quantities(self):
+        User = self.pool['res.users']
+        if self.quantity < 0:
```

Comment:
@leemannd Please use of float_compare as quantity is decimal precision aware https://github.com/odoo/odoo/blob/9e8f70e4849b0eeaca8b5cf51372ecfa23dc561b/addons/stock/wizard/stock_transfer_details.py#L139